pkg/controller/core/workload_controller.go

```diff
@@ -269,6 +269,10 @@ func (r *WorkloadReconciler) reconcileCheckBasedEviction(ctx context.Context, wl
 	log.V(3).Info("Workload is evicted due to admission checks")
 	workload.SetEvictedCondition(wl, kueue.WorkloadEvictedByAdmissionCheck, "At least one admission check is false")
 	err := workload.ApplyAdmissionStatus(ctx, r.client, wl, true)
+	if err == nil {
+		cqName, _ := r.queues.ClusterQueueFromLocalQueue(wl.Spec.QueueName)
+		metrics.ReportEvictedWorkloads(cqName, kueue.WorkloadEvictedByAdmissionCheck)
```

Comment:
Please cover in integration test that it is bumped.

Reply:
This one you could cover [here](https://github.com/kubernetes-sigs/kueue/blob/9b25d998b4cc317ab9539e55497be49fa19588d3/test/integration/controller/core/workload_controller_test.go#L382) with `util.ExpectEvictedWorkloadsTotalMetric(clusterQueue.Name, kueue.WorkloadEvictedByAdmissionCheck, 1)`.

However, it reveals a bug that `ClusterQueueFromLocalQueue` expects the namespace/LQ name. 
To fix that I suggest to create a function: `ClusterQueueForWorkload` which would call `ClusterQueueFromLocalQueue` for `fmt.Sprintf("%s/%s", wl.Namespace, wl.Spec.QueueName)`.

Then use this function here, and in the other place.

Reply:
There is already a func `ClusterQueueForWorkload`.
```go
func (m *Manager) ClusterQueueForWorkload(wl *kueue.Workload) (string, bool) {
	m.RLock()
	defer m.RUnlock()
	q, ok := m.localQueues[workload.QueueKey(wl)]
	if !ok {
		return "", false
	}
	_, ok = m.clusterQueues[q.ClusterQueue]
	return q.ClusterQueue, ok
}
```

Reply:
oh, this is even better then :)